odk/motech-settings.properties

```diff
@@ -0,0 +1,86 @@
+#Openid provider name
```

Comment:
I don't think we need these setting if this module is part of the modules build - the parent handles settings